exampleSite/content/posts/theme-documentation-basics/index.en.md

```diff
@@ -563,6 +563,17 @@ Please open the code block below to view the complete sample configuration :(far
         label = ""
         lightTheme = "github-light"
         darkTheme = "github-dark"
+      # {{< link "https://waline.js.org/en/" "Waline" >}} comment config
```

Comment:
Can you add also this shortcode: `{{< version 0.2.0 >}}`

The result will look something like this:

![image](https://user-images.githubusercontent.com/24962085/126205191-25bb23f8-12e5-48c9-b8ae-23e0197a8f8b.png)

Reply:
Done.